Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/FirebaseDatabase.kt

```diff
@@ -15,7 +15,8 @@ open class FirebaseDatabase {
         var roomNames: HashMap<String, Boolean>? = null
         var speakerIds: HashMap<String, Boolean>? = null
         var roomIds: HashMap<String, Boolean>? = null
-        var description: String? = null;
+        var primarySpeakerName: String? = null
```

Comment:
primarySpeakerName, description, name, and startTime should be non-null.
They were nullable in my denorm script because it pulled in the conferenceData into the same model it modified and wrote back out.

Reply:
updated